beets/util/__init__.py

```diff
@@ -731,6 +726,12 @@ def command_output(cmd, shell=False):
     This replaces `subprocess.check_output` which can have problems if lots of
     output is sent to stderr.
     """
+    # Couldn't use _convert_args here due to circular dep of beets.ui
+    for i, elem in enumerate(cmd):
+        if isinstance(elem, six.text_type):
+            import locale
+            encoding = locale.getdefaultlocale()[1] or 'utf8'
+            cmd[i] = elem.encode(encoding)
```

Comment:
I think the right solution here is probably to track down all the places where Unicode is passed to `command_output` and fix those. We could add an `assert isinstance(arg, bytes)` for each argument to the top here; that would help find the roots of the problem.

Except, of course, on Windows—where Unicode arguments should probably be tolerated.